bundles/com.zeligsoft.domain.dds4ccm/src/com/zeligsoft/domain/dds4ccm/utils/DDS4CCMMigrationModelTypeUtil.java

```diff
@@ -49,19 +48,14 @@
 	private static boolean migrateAll(Model model, boolean dryrun) {
 		boolean result = false;
 		
-		final Object rawValue = com.zeligsoft.base.zdl.util.ZDLUtil.getValue(
-				model, "DDS4CCM::DDS4CCM::DDS4CCMModel", "modelType");
-		if(!(rawValue instanceof EnumerationLiteral)){
-			return result;
-		}
-		EnumerationLiteral modelType = (EnumerationLiteral) rawValue;
+		DDS4CCMModel contextModel =  ZDLFactoryRegistry.INSTANCE.create(model, DDS4CCMModel.class);
+		ModelTypeEnum modelType = contextModel.getModelType();
 		
-		if(modelType.getName().equals("ATCD")){
+		if(modelType.equals(ModelTypeEnum.ATCD)){
```

Comment:
FYI: Enumeration literals also compare with ==, so `modelType == ModelTypeEnum.ATCD` would have been sufficient. But, there is no point in making additional changes.